Runtime/Highlight/DefaultHighlighter.cs

```diff
@@ -0,0 +1,445 @@
+﻿using System;
+using System.Linq;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.Rendering;
+using Innoactive.Creator.Unity;
+
+namespace Innoactive.Creator.BasicInteraction
+{
+    /// <summary>
+    /// Adds highlighting functionality to a GameObject with Renderers.
+    /// </summary>
+    /// <remarks>
+    /// Highlights are always queued following a LIFO (Last In First Out) scheme. 
+    /// </remarks>
+    [DisallowMultipleComponent]
+    public class DefaultHighlighter : MonoBehaviour
```

Comment:
I am not sure if this works or if we need a layer underneath like an abstract `BaseHighlighter` or a `IHighlightable` or something like that. If you look at something like the Teleporter which requires a special way of highlighting (changing texture color not making each mesh a big fat bloop of color) but at the same you add the DefaultHighlighter in this PR https://github.com/Innoactive/Creator-Pro/pull/56 it will clash. Also if somebody wants to create their own Highlighter this default one without an interface will most likely make things complicated